### ISSUE TITLE
Term issue: NTR:add proteins, proproteins, degredation products, or protein complexes used in clinical tests

### ISSUE THREAD

**Issue by LCCarmody** [PROconsortium/PRoteinOntology#181]
I was hoping that several proteins, proproteins, degradation products, and/or protein complexes could be added. Our need is to reference proteins and biomarkers used in clinical tests.  
Here is the Updated list of proteins (or protein products) we initially intended to include.  

Our google doc is [HERE](https://docs.google.com/spreadsheets/d/1cNiFyVWdWN6raKhKtc1ehtb5EPmJZ3nTqynR5_M7--0/edit#gid=1345880538) and this references our [Github ticket](https://github.com/monarch-initiative/MAxO/issues/168).

Many thanks. 

Term requested | PRO label | PRO ID | MAXO need
-- | -- | -- | --
D-dimer |   |   | Fibrin degradation fragment test
procalcitonin |   |   | procalcitionin level test
proinsulin |   |   | proinsulin measurement, proinsulin to insulin ratio measurement
insulin | insulin (human) | PR_P01308 | insulin level, proinsulin measurement, proinsulin to insulin ratio measurement
creatinine |   |   | creatinine level test
Ferritin |   | Need the multi protein complex of heavy/light ferritin | ferritin level test, ferritin test
albumin | Serum albumin | PR_P02768 | albumin level test, albumin to creatinine ration level test
lactic acid dehydrogenase/ lactate dehydrogenase |   |   | lactate dehydrogenase test https://www.mayocliniclabs.com/test-catalog/Overview/8344
creatine kinase | creatine kinase | PR_000029971* This is not human specific, but would like a human specific version and not the individual subunits. | creatine kinase level test
Aspartate aminotransferase | asparate aminotransferase | PR_P17174 | Aspartate aminotransferase levels
alanine transaminase |   | Need a combined term for human alanine aminotransferase | alanine transaminase test, ALT test
granulocyte colony-stimulating factor | granulocyte colony-stimulating factor (human) | PR_P09919 | G-CSF test, G-CSF treatment
Adenine phosphoribosyltransferase | Adenine phosphoribosyltransferase (human) | PR_P07741 | Adenine phosphoribosyltransferase (APRT) activity test
Cystatin-C | cystatin-C (human) | PR_P01034 | cystatin-C level test
protein | protein | PR_000000001 | protein in urine test
alkaline phosphatase | alkaline phosphatase, tissue-nonspecific | PR_000003968 | alkaline phosphatase levels test

**Comment by nataled** [comment]
Hi Leigh,

It wasn't explicitly stated for some of these, but are you looking specifically for human versions for *all* of these terms?

**Comment by LCCarmody** [comment]
MAXO is ontology is designed for clinical use for human disease, though I suppose, theoretically, it could be used for veterinary medicine.  So, either all terms should be human specific or all terms should non-species specific.  I think it would be easier and make most sense to use all human.

**Comment by nataled** [comment]
Apologies, but as I work through these I'm going to have a number of questions. The vast majority of the tests don't have information about what exactly is being tested (at least that I can find online), so it's difficult to know what is needed in those cases where there are multiple options. For example:

1) alkaline phosphatase: there are several types in humans. From what I can find, it seems that the tissue non-specific type (the one you indicated) is indeed the one of interest, since that seems to be the predominant form in blood. Can you confirm? Also, the form in blood seems to be a homodimer. Does the test care about that?

2) aspartate aminotransferase: there are three isozymes in human. One is mitochondrial, the other two are cytoplasmic. Do you need a term that combines these in some way, or a subset of them? I see that you specifically requested a combined term for alanine aminotransferase.

3) creatine kinase: I surmise from your request that you want the creatine kinase complex, but there are multiple versions (based on a quick look at Wikipedia: https://en.wikipedia.org/wiki/Creatine_kinase#Types) and it seems like the test will find any of those versions that are found in blood. By any chance do you know which ones to include in what I presume should be a 'set'-type of PRO term?

**Comment by LCCarmody** [comment]
I will try to answer all, but will have to get back to you with more detailed sources.

**Comment by LCCarmody** [comment]
1. I think the test measures the activity of all Alkaline phosphatase, including TNAP, IAP, PAP, LAP. So, I think the term would be a Alkaline Phosphatase family of proteins. "When used as part of a typical comprehensive metabolic panel, this test represents the sum of all soluble AP isoenzyme activity in serum or plasma. " PMID: [30284677](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6351214/)

**Comment by nataled** [comment]
I will make the term for the four isotypes, but I'm not sure it is really what you need. I note this sentence in section 3.1 of the paper you reference: "The standard clinical chemistry AP tests used in laboratory medicine report total AP enzyme activity rather than the amount of total AP protein."

Given that, I suspect what you really need is http://purl.obolibrary.org/obo/GO_0004035 (alkaline phosphatase activity). Just wanted to alert you.

**Comment by LCCarmody** [comment]
Thanks. We will also add an activity for this and others where necessary, but that work hasn't begun yet.

2. Based on MAYO clinic description (https://www.mayocliniclabs.com/test-catalog/Clinical+and+Interpretive/8360) I believe it measures all 3, but the predominate forms measured are cytoplasmic.

**Comment by LCCarmody** [comment]
3.  Using the MAYO clinic interpretation section, It appears to be several different complexes (https://www.mayocliniclabs.com/test-catalog/Clinical+and+Interpretive/35063).

CK-MB and CK-BB and macro CK are found much more often than>>>Macro CK type II, but the test appears to measure them all.

**Comment by nataled** [comment]
With respect to #2 (aspartate aminotransferase), upon further reading, I find that one of the isozymes I mentioned does not actually work as an aspartate aminotransferase. Since the test is again one of activity measurement rather than protein measurement, I will create a term that only accounts for the two active versions.

**Comment by nataled** [comment]
I have completed the work. I placed the mapping in context of your spreadsheet (attached), and also listed the identifiers and names below. The identifiers can be used right away, and the PURLs will resolve. However, the terms themselves won't become part of the downloadable PRO file until the next release, which won't be for a month or so. Let me know if you need the stanzas (they are in OBO format currently). Just FYI, I created a good deal more terms than the 15 shown below (about 45 in all) in order to make the ones that needed to be created (others already existed).

Looking at the spreadsheet, make note of the color coding (key at bottom).

Thank you very much for the request, and please do let us know if you have further requests.

PR:000050369 | D-dimer (human)
-- | --
PR:000050338 | procalcitonin   (human)
PR:000048036 | proinsulin   (human)
PR:000050340 | insulin   hormone (human)
NA | creatinine (not a protein)
PR:000050342 | ferritin complex   (human)
PR_P02768 | serum   albumin (human)
PR:000050367 | lactate   dehydrogenase complex (human)
PR:000050361 | creatine   kinase complex (human)
PR:000050353 | aspartate   aminotransferase (human)
PR:000050341 | alanine   aminotransferase (human)
PR_P09919 | granulocyte   colony-stimulating factor (human)
PR_P07741 | adenine   phosphoribosyltransferase (human)
PR_P01034 | cystatin-C   (human)
PR_000029067 | Homo   sapiens protein
PR:000050352 | alkaline   phosphatase, active form (human)



[Copy of NTR_ preprocessed proteins and degredation products.xlsx](https://github.com/PROconsortium/PRoteinOntology/files/4630340/Copy.of.NTR_.preprocessed.proteins.and.degredation.products.xlsx)

**Comment by LCCarmody** [comment]
That's fantastic!  I will definitely look to see the additional terms.

**Comment by LCCarmody** [comment]
If you mean by stanzas the full OBO serialization of the term you added, that would be great

**Comment by nataled** [comment]
Because the terms you requested are present only in an intermediate release, you won't be able to search for them yet. When it goes public, you would be able to search the PRO web site (purl.obolibrary.org/obo/pr) for "MAxO" to find them all.

By "stanza" I mean this: https://proconsortium.org/cgi-bin/entry_pro?stanza_id=PR:000050352

I can provide a file that contains everything tagged with "MAxO" if that helps, though it might miss a few of the pre-existing terms.

**Comment by LCCarmody** [comment]
Yes, that would be helpful.  I understand it won't be searchable until the release, but I will integrate these terms and set up a PR import.  

Hmm.  I will ask @matentzn to deal with OBO vs OWL.

**Comment by matentzn** [comment]
@nataled by "I trust you know how to deal with those?" I assume you mean replacing `_` with `:`. 

If we know the ids, we dont really need anything else! When do you think these terms will appear in production (pr.owl)?

**Comment by nataled** [comment]
@matentzn yes, the replacement of delimiter. We had a release just a few weeks ago, so another won't come out for at least another month or two.

@LCCarmody I've attached the OBO file with all the terms made for this request. I attempted to trace back to root for the terms that matter (if you open the file in Protege, you'll notice that some identifiers were not fully fleshed out, but these are mostly genes and a few miscellaneous bits needed to define some of the terms). Note that I had to change the extension to get github to accept it.
[PR_terms_for_MAxO.obo.txt](https://github.com/PROconsortium/PRoteinOntology/files/4630963/PR_terms_for_MAxO.obo.txt)
